tests/integrations/falcon/test_falcon.py

```diff
@@ -380,17 +380,20 @@ def test_does_not_leak_scope(sentry_init, capture_events):
     sentry_init(integrations=[FalconIntegration()])
     events = capture_events()
 
-    Scope.get_isolation_scope().set_tag("request_data", False)
+    scope = Scope.get_isolation_scope()
+    scope.set_tag("request_data", False)
```

Comment:
Why are this change, and the rest of the changes in this file, necessary? The code seems like it would function identically. And, the code we are replacing already follows the new API

Reply:
Yeah ignore this file, I'll remove it. I messed something up when splitting up the changes.